db/03_demo_data.sql

```diff
@@ -0,0 +1,117 @@
+('da3d5336-0c5f-4df4-a2a0-646750f8739d','13.6552','9.0256','Real Hospital','http://www.google.org/','Clinical',NULL,0,0,0,'Not me','jkl@mno.com','','Not my site',SUBDATE(CURDATE(), 30*21),SUBDATE(CURDATE(), 30*21),'test',1,'',NULL),
+('7ed46521-5e48-4b42-96f3-1bfb837ca345','30.6552','59.0256','Téstè mañana','http://www.google-test.org/','Clinical',NULL,0,0,0,'Neo','mno@pqr.com','','Neos site',SUBDATE(CURDATE(), 30*27),SUBDATE(CURDATE(), 30*27),'testadmin',1,'',NULL);
+/*!40000 ALTER TABLE `atlas` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Dumping data for table `auth`
+--
+
+LOCK TABLES `auth` WRITE, `atlas` WRITE;
+/*!40000 ALTER TABLE `auth` DISABLE KEYS */;
+INSERT INTO `auth`(`atlas_id`,`principal`,`token`,`privileges`,`expires`) SELECT `id`,`created_by`,NULL,'ALL',NULL FROM `atlas`;
+/*!40000 ALTER TABLE `auth` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Dumping data for table `distributions`
+--
+
+LOCK TABLES `distributions` WRITE;
+/*!40000 ALTER TABLE `distributions` DISABLE KEYS */;
+INSERT INTO `distributions` VALUES (1,'OpenMRS 1.x','2016-05-25 13:35:49',1),(2,'Reference Application 2.x','2016-05-25 13:35:49',1),(3,'Bahmni','2016-05-25 13:35:49',1),(4,'KenyaEMR','2016-05-25 13:35:49',1),(6,'others','2016-07-13 18:20:58',0);
+/*!40000 ALTER TABLE `distributions` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+--
+-- Dumping data for table `migrations`
+--
+
+LOCK TABLES `migrations` WRITE;
+/*!40000 ALTER TABLE `migrations` DISABLE KEYS */;
+INSERT INTO `migrations` VALUES ('2014_05_20_085005_create_atlas',1),('2014_05_20_093740_create_auth',1),('2014_05_23_205713_update_archive',1),('2014_05_24_082932_change_column_size',1),('2014_05_24_104445_create_admin',1),('2014_05_28_060735_rename_created_by',1),('2014_06_23_213056_add_show_counts',1),('2014_07_09_165540_drop_archive_datechanged_column',1),('2014_07_15_060605_add_uuid_and_action_to_archive',1),('2014_07_31_065930_add_openmrs_version_column_in_atlas',1),('2016_01_22_153112_create_distributions',1),('2016_02_17_170405_insert_distributions',1);
+/*!40000 ALTER TABLE `migrations` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Dumping data for table `types`
+--
+
+LOCK TABLES `types` WRITE;
+/*!40000 ALTER TABLE `types` DISABLE KEYS */;
+INSERT INTO `types` VALUES (1,'Research','https://maps.google.com/intl/en_us/mapfiles/ms/micons/green-dot.png'),
+(2,'Clinical','https://maps.google.com/intl/en_us/mapfiles/ms/micons/purple-dot.png'),
+(3,'Development','https://maps.google.com/intl/en_us/mapfiles/ms/micons/blue-dot.png'),
+(4,'Evaluation','https://maps.google.com/intl/en_us/mapfiles/ms/micons/yellow-dot.png'),
+(5,'Other','https://maps.google.com/intl/en_us/mapfiles/ms/micons/red-dot.png');
+/*!40000 ALTER TABLE `types` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Dumping data for table `types`
+--
+
+LOCK TABLES `versions` WRITE;
+/*!40000 ALTER TABLE `versions` DISABLE KEYS */;
+INSERT INTO `versions` VALUES (1,'2.1'),
+(2,'2.1-SNAPSHOT'),
```

Comment:
This should be a list of OpenMRS versions, not Atlas module versions. I would suggest values like 1.9, 1.10, 1.11, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, and 2.9.